mod/display.php

```diff
@@ -499,3 +499,12 @@ function display_content(App $a, $update = 0) {
 	return $o;
 }
 
+function displayShowFeed($item_id) {
```

Comment:
this function should have a doxygen header ;-)

Reply:
Oops, yeah. Typical early morning coding. But please praise me anyway for writing the the function name correctly.

Reply:
You are right. I praise you ;-)